Source/WTF/wtf/text/StringView.h

```diff
@@ -588,9 +588,8 @@ inline StringView::UpconvertedCharacters::UpconvertedCharacters(StringView strin
     }
     const LChar* characters8 = string.characters8();
     unsigned length = string.m_length;
-    m_upconvertedCharacters.reserveInitialCapacity(length);
-    for (unsigned i = 0; i < length; ++i)
-        m_upconvertedCharacters.uncheckedAppend(characters8[i]);
+    m_upconvertedCharacters.resize(length);
```

Comment:
I think this zeroes out all the characters before overwriting them with the loop below. Could we get even better performance by finding a way to do it without zeroing them? Also, we normally use `grow` rather than `resize` when we are starting with an empty vector to avoid a wasted branch.

Reply:
Actually, we do not zeroed out these memory because `UChar` is POD :)
```
  1178 template<typename T, size_t inlineCapacity, typename OverflowHandler, size_t minCapacity, typename Malloc>
  1179 inline void Vector<T, inlineCapacity, OverflowHandler, minCapacity, Malloc>::resize(size_t size)
  1180 {
  1181     if (size <= m_size) {
  1182         TypeOperations::destruct(begin() + size, end());
  1183         asanBufferSizeWillChangeTo(size);
  1184     } else {
  1185         if (size > capacity())
  1186             expandCapacity<FailureAction::Crash>(size);
  1187         asanBufferSizeWillChangeTo(size);
  1188         if (begin())
  1189             TypeOperations::initializeIfNonPOD(end(), begin() + size);
  1190     }
  1191
  1192     m_size = size;
  1193 }
  ```
  L1189 clears vector only when it is non POD.